examplejava/src/main/java/io/matchmore/sdk/examplejava/MainActivity.java

```diff
@@ -37,7 +37,7 @@ protected void onCreate(Bundle savedInstanceState) {
         // Creating main device.
         matchMore.startUsingMainDevice(device -> {
```

Comment:
Maybe you want to change it in your example as well. 

Let's always go for matchmore with little m for more